src/components/OwnerPanel/styles.module.scss

```diff
@@ -88,3 +88,9 @@ $container-size: 160px;
   background-color: #dd678a;
   border: 1px solid #dd678a;
 }
+
+p {
```

Comment:
Instead of styling the actual tag, create a specific class for this so that you don't end up changing all of the nested p tags